matrix/matrix_binary_search.py

```diff
@@ -0,0 +1,54 @@
+def binary_search(array: list, lower_bound: int, upper_bound: int, value: int) -> bool:
+    """
+    This function carries out Binary search on a 1d array
+    Array: A 1d sorted array
+    lower_bound: the lowest index for the binary search algorithm
+    upper_bound: the highest index for the binary search algorithm
+    value : the value ment to be searched
+
+    matrix = [1, 4, 7, 11, 15]
+    
+    target = 1
+
+    >>> binary_search(matrix, 0, len(matrix)-1, 1)
+    True
+    """
+    r = int((lower_bound+upper_bound)/2)
+    
+    if (array[r] == value):
+        
+        return True
+    if((lower_bound == r) or (upper_bound == r)):
+        return False
+    if (array[r] < value):

```

Comment:
As discussed in CONTRIBUTING.md, please run this code thru psf/black to get rid of all these extra parentheses.  This ain't JavaScript.